src/v2/ion/uiSchemaTransformer.js

```diff
@@ -60,6 +60,7 @@ const createUISchema = (transformedResp, remediationForm, settings) => {
   return remediationValue.map(ionFormField => {
     const uiSchemaDefaultConfig = {
       'label-top': true,
+      'multirowError': true,
```

Comment:
https://okta.box.com/s/d3us4gwdzc81bnfy9uuabgwcdomfkpn6
here is how the final output looks like, the error messages in this picture will be updated to the new messages below